backends/ebpf/runtime/pcap_util.h

```diff
@@ -0,0 +1,207 @@
+/*
+Copyright 2018 VMware, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+ * A pcap utility library which provides several functions to read from and
+ * write packets to pcap files.
+ */
+
+#ifndef BACKENDS_EBPF_RUNTIME_EBPF_PCAP_UTIL_H_
+#define BACKENDS_EBPF_RUNTIME_EBPF_PCAP_UTIL_H_
+
+#define PCAP_DONT_INCLUDE_PCAP_BPF_H
+#include <pcap/pcap.h>
+#include <stdint.h>     // uint32_t, uint16_t
+
+/* Interfaces are named by integers */
+typedef uint16_t iface_index;
```

Comment:
This comment is still unaddressed.

Reply:
My intention was to limit the amount of possible files to 65535 for simplicity and sanity. I do a check for that in the runtime. Pcap files can be much larger, so I leave the indexing at uint64_t to avoid overflows.